### ISSUE TITLE
ibazel can't find tsconfig

### ISSUE THREAD

**Issue by appsforartists** [bazelbuild/rules_typescript#121]
```
❯ ibazel run //packages/playground
Line: packages/playground/src/index.ts
Line: packages/core/src/index.ts
Line: /tsconfig.json
Error watching: lstat /tsconfig.json: no such file or directory
```

```python
# //packages/playground/BUILD
package(
  default_visibility = ["//visibility:public"],
)

load("@build_bazel_rules_typescript//:defs.bzl", "ts_devserver", "ts_library")

ts_library(
  name = "playground",
  module_name = "dgitize-playground",
  module_root = "src",
  srcs = glob(
    include = ["src/**/*.ts"],
    exclude = ["**/__tests__/*"],
  ),
  deps = ["//packages/core"],
  tsconfig = "//:tsconfig.json",
)

ts_devserver(
  name = "devserver",
  deps = [":playground"],
  serving_path = "/dist/bundle.js",
  static_files = ["site/index.html"],
)


# //WORKSPACE
workspace(name = "dgitize_root")

load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")

git_repository(
  name = "build_bazel_rules_nodejs",
  remote = "https://github.com/bazelbuild/rules_nodejs.git",
  tag = "0.3.1",
)

load("@build_bazel_rules_nodejs//:defs.bzl", "node_repositories")

node_repositories(package_json = ["//:package.json"])

git_repository(
  name = "build_bazel_rules_typescript",
  remote = "https://github.com/bazelbuild/rules_typescript.git",
  tag = "0.7.1",
)

load("@build_bazel_rules_typescript//:setup.bzl", "ts_setup_workspace")

ts_setup_workspace()

# Go rules needed for ts_devserver
http_archive(
  name = "io_bazel_rules_go",
  url = "https://github.com/bazelbuild/rules_go/releases/download/0.8.1/rules_go-0.8.1.tar.gz",
  sha256 = "90bb270d0a92ed5c83558b2797346917c46547f6f7103e648941ecdb6b9d0e72",
)

load("@io_bazel_rules_go//go:def.bzl", "go_rules_dependencies", "go_register_toolchains")

go_rules_dependencies()
go_register_toolchains()
```

**Comment by alexeagle** [comment]
What version of bazel and ibazel?
Content of the root BUILD file?

**Comment by appsforartists** [comment]
```
//BUILD
package(
  default_visibility = ["//visibility:public"],
)

exports_files(["tsconfig.json"])

# NOTE: this will move to node_modules/BUILD in a later release
filegroup(
  name = "node_modules",
  srcs = glob([
    "node_modules/**/*.js",
    "node_modules/**/*.d.ts",
    "node_modules/**/*.json",
  ])
)
```

bazel 0.5.4

Trying to find the ibazel version…

**Comment by appsforartists** [comment]
Fixed by upgrading to bazel@0.9 and ibazel@0.2.0 (with `yarn add @bazel/ibazel`).

```
❯ yarn run ibazel run //packages/playground:devserver
```
works.

Thanks for your help!

**Comment by alexeagle** [comment]
Cool - I recommend adding a bazel version check in your `WORKSPACE` so other developers on the project don't fall into a hole: https://github.com/angular/angular/blob/cdc66f616467e8623e04b49c2a029ba207a073bd/WORKSPACE#L13
